lib/agg_regrid/__init__.py

```diff
@@ -0,0 +1,420 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of iris-extras.
+#
+# iris-extras is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# iris-extras is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with iris-extras.  If not, see <http://www.gnu.org/licenses/>.
+"""A package for experimental regridding functionality."""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, xrange, zip)
```

Comment:
I don't like this `xrange`; it encourages writing outdated calls when Python 3 passes when it shouldn't.